yarn-project/end-to-end/src/spartan/k8_utils.ts

```diff
@@ -0,0 +1,222 @@
+import { createDebugLogger } from '@aztec/aztec.js';
```

Comment:
nit: I'm in the Charlie camp of 'X_utils.ts is not more descriptive than X.ts' but YMMV